src/integrations/serde.rs

```diff
@@ -1,13 +1,42 @@
 use serde::{de, ser};
 use serde::ser::SerializeMap;
-use std::collections::HashMap;
 use std::fmt;
+use std::collections::HashMap;
 
 use ::{GraphQLError, Value};
 use ast::InputValue;
 use executor::ExecutionError;
 use parser::{ParseError, Spanning, SourcePosition};
 use validation::RuleError;
+use serde_json::Value as Json;
+
+
+impl InputValue {
+    /// Converts serde_json::Value to juniper::InputValue
+    pub fn from_json(json: Json) -> InputValue {
```

Comment:
[Deserialization is already implemented for `InputValue`](https://github.com/mhallin/juniper/blob/032b03add334f230854223bb69dcde1568ecb075/src/integrations/serde.rs#L52-L119). Why not use it?